feature/platform/tests/power_admin_down_up_test/metadata.textproto

```diff
@@ -30,3 +30,12 @@ platform_exceptions: {
     skip_controller_card_power_admin: true
   }
 }
+platform_exceptions: {
+  platform: {
+    vendor: CISCO
+  }
+  deviations: {
+    leaf_ref_validation: true
```

Comment:
leaf_ref_validation this is a generic name, Can the deviation name be specific to this oc config.?

Reply:
actually i kept it generic on purpose; because there are some other tests where we need to do a leaf reference; Like some of the bgp config path specifically. Otherwise we keep it specific to power related on all components;

Reply:
Can you please share the details of the oc leaves

cc: @dplore please share your inputs

Reply:
Have change the deviaion name to power_enable_disable_leaf_ref_validation; The changes recommended is now fixed;